homeassistant/components/plugwise/entity.py

```diff
@@ -22,50 +15,33 @@ class PlugwiseEntity(CoordinatorEntity[PlugwiseData]):
 
     def __init__(
         self,
-        api: Smile,
         coordinator: PlugwiseDataUpdateCoordinator,
-        name: str,
-        dev_id: str,
+        device_id: str,
     ) -> None:
         """Initialise the gateway."""
         super().__init__(coordinator)
+        self._dev_id = device_id
 
-        self._api = api
-        self._name = name
-        self._dev_id = dev_id
-        self._entity_name = self._name
-
-    @property
-    def name(self) -> str | None:
-        """Return the name of the entity, if any."""
-        return self._name
-
-    @property
-    def device_info(self) -> DeviceInfo:
-        """Return the device information."""
-        data = self.coordinator.data.devices[self._dev_id]
-        device_information = DeviceInfo(
-            identifiers={(DOMAIN, self._dev_id)},
-            name=self._entity_name,
-            manufacturer="Plugwise",
-        )
-
+        configuration_url: str | None = None
         if entry := self.coordinator.config_entry:
-            device_information[
-                ATTR_CONFIGURATION_URL
-            ] = f"http://{entry.data[CONF_HOST]}"
-
-        if model := data.get("model"):
-            device_information[ATTR_MODEL] = model
+            configuration_url = f"http://{entry.data[CONF_HOST]}"
+
+        data = self.coordinator.data.devices[device_id]
```

Comment:
Is `self` needed here? Lower down `self` is not used.

Reply:
Nope, can be removed, left over from moving these lines from the property method to the constructor. 👍

Reply:
Adjusted in [bbf3e3b](https://github.com/home-assistant/core/pull/66074/commits/bbf3e3bc737f7e1c35a976f278f681a009e76eef)